source/includes/fundamentals/code-examples/ExtendedJson.cs

```diff
@@ -14,6 +14,19 @@ public static void Main(string[] args)
             Console.WriteLine(document.ToJson());
             // end-read-ejson
         }
+        
+        {
+            // start-read-ejson-reader
+            var ejson = "{\n\"_id\": { \"$oid\": \"573a1391f29313caabcd9637\" },\n  \"createdAt\": { \"$date\": { \"$numberLong\": \"1601499609\" }},\n\"numViews\": { \"$numberLong\": \"36520312\" }\n}\n\n";
+            var subject = new BsonDocumentSerializer();
```

Comment:
The variable name `subject` is sometimes used when writing tests, where it stands for "the subject under test".

Not sure the name `subject` makes any sense here. I would call this variable `serializer`.

But see comment below about whether we even want this example.